hackage-server.cabal

```diff
@@ -349,8 +350,8 @@ library lib-server
   build-depends:
     , HStringTemplate       ^>= 0.8
     , HTTP                  ^>= 4000.3.6
-    , QuickCheck            ^>= 2.9
-    , acid-state            ^>= 0.14
+    , QuickCheck            > 2.9 && < 2.15
```

Comment:
The `>` explicitly excludes version `2.9`. I suspect all lower bounds should been `>=`?

Yet, I'd suggest supporting only single major version, i.e. in this case `^>=2.14` (or even `^>=2.14.2` to go for the latest). Older and in between versions are probably never tested anyway.